src/test/java/apoc/export/cypher/ExportCypherTest.java

```diff
@@ -41,7 +47,11 @@
             "schema await%n");
 
     private static final String EXPECTED_RELATIONSHIPS = String.format("begin%n" +
-            "MATCH (n1:`UNIQUE IMPORT LABEL`{`UNIQUE IMPORT ID`:0}), (n2:`Bar`{`name`:\"bar\"}) CREATE (n1)-[:`KNOWS`]->(n2);%n" +
+            "MATCH (n1:`UNIQUE IMPORT LABEL`{`UNIQUE IMPORT ID`:0}), (n2:`Bar`{`name`:\"bar\"}) CREATE (n1)-[n:`KNOWS` {`since`:2016}]->(n2);%n" +
+            "commit%n");
+
+    private static final String EXPECTED_RELATIONSHIPS_MERGE = String.format("begin%n" +
+            "MATCH (n1:`UNIQUE IMPORT LABEL`{`UNIQUE IMPORT ID`:0}), (n2:`Bar`{`name`:\"bar\"}) MERGE (n1)-[n:`KNOWS`]->(n2) SET n.`since`=2016;%n" +
```

Comment:
Perhaps we have to define the "high level" operations better:

* create - all CREATE
* add structure -> match for nodes + merge for rels
* update structure -> merge + match for nodes and rels 
* update all -> SET instead of ON CREATE SET